.github/workflows/build-plugin-archive.yml

```diff
@@ -0,0 +1,241 @@
+name: Create plugin archive
+
+on:
+  workflow_call:
+    inputs:
+      PLUGIN_MAIN_FILE:
+        description: The path/name of the plugin main file.
+        required: false
+        default: './index.php'
+        type: string
+      PLUGIN_VERSION:
+        description: The plugin version.
+        required: true
+        type: string
+      PLUGIN_BASENAME:
+        description: The base name of the resulting zip archive. Falls back to the repository name.
+        default: ''
+        required: false
+        type: string
+      COMPOSER_ARGS_DEV:
+        description: Set of arguments passed to Composer for build setup.
+        default: '--prefer-dist'
+        required: false
+        type: string
+      COMPOSER_ARGS:
+        description: Set of arguments passed to Composer for production setup.
+        default: '--no-dev --no-scripts --prefer-dist --optimize-autoloader'
+        required: false
+        type: string
+      NPM_REGISTRY_DOMAIN:
+        description: Domain of the private npm registry.
+        default: https://npm.pkg.github.com/
+        required: false
+        type: string
+      NODE_VERSION:
+        description: Node version to use for asset compilation.
+        default: 16
+        required: false
+        type: string
+      PHP_VERSION:
+        description: PHP version to use during packaging.
+        default: 8.2
+        required: false
+        type: string
+      EXTRA_PHP_FILE:
+        description: Path to a custom php script to run before creating the release archive.
+        default: ''
+        required: false
+        type: string
+      EXTRA_PHP:
+        description: Run custom php code before creating the release archive.
+        default: ''
+        required: false
+        type: string
+    secrets:
```

Comment:
Please add the `PACKAGE_MANAGER` input as seen e.g. here: https://github.com/inpsyde/reusable-workflows/blob/f362c32c9d14f1c27f1586bb1a13a44b5a2e3ab4/.github/workflows/static-analysis-js.yml#L21C7-L25

Reply:
Added